api/package.json

```diff
@@ -16,12 +16,15 @@
     "fastify": "4.17.0",
     "fastify-auth0-verify": "^1.0.0",
     "fastify-plugin": "^4.3.0",
+    "jsonwebtoken": "9.0.0",
+    "mongodb": "4",
```

Comment:
I added `bson` instead of `mongodb`, because I thought it is smaller. Also, I wonder what coupling this would add between prisma and our package.

Reply:
I did, initially, but then I realised we already have `mongodb` as a peer dep of connect mongo :man_shrugging:

This way it's explicit and we get to use it.

> what coupling this would add between prisma and our package

None, prisma does its own thing. I can only see there being an issue if ObjectID changed and... I really hope they don't!